index.rst

```diff
@@ -113,10 +113,14 @@ It should have a markdown cell that provides explanations and/or links to extern
 All package imports must be done in the first code cell.
 
 The second subsection should always be ``1.2 Define Functions and Parameters``.
-Globally defined utility functions, plotting defaults, or constants should be here.
-It is acceptable to rename the subsection to be more specific to the notebook, and/or to use sub-subsections like
-``1.2.1 Define global cosmological parameter values`` or ``1.2.2 Define a function to make an image cutout``.
-If applicable, it is preferred to leave this subsection header in the document and state that no additional functions and parameters are used.
+Globally defined utility functions, classes, plotting defaults, or constants should be here.
+
+If a notebook has no functions or parameters to define, it is preferred to leave this subsection header in the document
+and state that no additional functions and parameters are used.
+
+If a notebook has many such things to define, it is acceptable to rename the subsection to be more specific to the notebook,
+and/or to use sub-subsections like ``1.2.1 Define global cosmological parameter values`` or ``1.2.2 Define a function to make an image cutout``.
+
```

Comment:
At this point, should we warn the reader about the exception below (e.g., "see section "function and classes" below")

Reply:
implemented, thank you!